app/controllers/declaration/AdditionalInformationController.scala

```diff
@@ -45,14 +45,15 @@ class AdditionalInformationController @Inject()(
   mcc: MessagesControllerComponents,
   additionalInformationPage: additional_information
 )(implicit ec: ExecutionContext)
-    extends FrontendController(mcc) with I18nSupport with ModelCacheable {
+    extends FrontendController(mcc) with I18nSupport with ModelCacheable with SubmissionErrors {
 
   val elementLimit = 99
 
   def displayPage(mode: Mode, itemId: String): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
+    val frm = form().withSubmissionErrors()
```

Comment:
The same here

Reply:
Addressed this one as an example.  Not sure its worth it though.